srcpkgs/gcr4/template

```diff
@@ -0,0 +1,43 @@
+# Template file for 'gcr4'
+pkgname=gcr4
+version=4.0.0
+revision=1
+build_style=meson
+build_helper="gir"
+configure_args="$(vopt_bool gir introspection) $(vopt_bool gir gtk_doc)"
+hostmakedepends="gettext glib-devel gnupg gi-docgen gettext pkg-config openssh
+ libxslt $(vopt_if gir vala)"
+makedepends="libgcrypt-devel libsecret-devel p11-kit-devel libxslt-devel gtk4-devel"
+depends="desktop-file-utils hicolor-icon-theme gnupg"
+checkdepends="dbus"
+short_desc="GNOME crypto package"
+maintainer="Orphaned <orphan@voidlinux.org>"
```

Comment:
Don't know if this is normal but to me it looks a bit weird pushing new packages that are already orphaned ^^

Reply:
I adapted this template from gcr which was orphaned. @oreo639 could you assign me as the maintainer?

Reply:
Sure.